reboot-tool.c

```diff
@@ -20,10 +20,11 @@ int main(int argc, char *argv[])
     char *str1 = "bootloader";
     char *str2 = "recovery";
     char *str3 = "dnx";
+    char *str4 = "autodloader";
```

Comment:
nit. This is starting to look like "array of strings could be handy". Maybe the next time ;-)

Reply:
yeah :)